docs/sources/panels-visualizations/visualizations/_index.md

```diff
@@ -20,10 +20,14 @@ weight: 10
 
 Grafana offers a variety of visualizations to support different use cases. This section of the documentation highlights the built-in visualizations, their options and typical usage.
 
+{{< youtube id="JwF6FgeotaU" >}}
+
 {{% admonition type="note" %}}
 If you are unsure which visualization to pick, Grafana can provide visualization suggestions based on the panel query. When you select a visualization, Grafana will show a preview with that visualization applied.
 {{% /admonition %}}
 
+{{< youtube id="yNRnLyVntUw" >}}
```

Comment:
I think it makes more sense logically to move this video to after the categorized list of visualizations because it gives the reader a chance to learn about the different types of visualizations and use cases in text first. I think it would then be helpful to have a lead-in sentence like

> The following video shows you how to create visualizations to match your needs:

Also moving the video breaks up the video/note/video chunk that looks a bit odd in preview.

Reply:
Thank you so much for your recommendations, @imatwawana ! Sounds great. 
Please let me know if there is anything I should do on my end.

Reply:
Hey @LisaHJung, if you can go ahead and implement that suggestion, that would be great. You'll want to move the video down to after the list that ends [here](https://github.com/grafana/grafana/pull/83928/files#diff-6a26b5122b603a71deff5b36a1410210731c18b46ac3a7b79525db511c87e1c7L52) and add the intro line I suggested. When you've done that, go ahead and ping me to review again.

Reply:
Hey @imatwawana ! I just committed the suggested change. :)

Reply:
@imatwawana , one more thing! This video specifically covers how to create gauge, time series line graph, stats, logs, and node graph visualizations. Should we mention that in the introductory sentence?

Reply:
Sure, go ahead and update that intro. It's good for the reader to know what they're going to get by watching the video before they start.

Reply:
Wonderful! Just updated the intro @imatwawana. Could we merge it now?